services/prefect/prefect-agent.service

```diff
@@ -3,6 +3,7 @@ Description=prefect agents
 
 [Service]
 Environment="DATAPATH=/home/sglyon/scraper-outputs"
+Environment="COVID_DB_CONN_URI=<postgress-url>"
```

Comment:
This was on the system but not in github, so I'm adding it...  though perhaps we can just delete this file if Prefect Cloud works out. 🤷